### PR TITLE
Add script for GPL

### LINKED ISSUE #1305: Update all the GPL headers
All of the GPL headers need to say the current year (2016) and files that don't have GPL headers need to have GPL headers.

### PR DESCRIPTION
**Pull request by va6996**
Instead of manually fixing everything, I created a script to do the same. It can be tweaked  as per requirement. Let me know it there needs to be any changes.
Fixes #1305

### CODE REVIEW

File: TShockAPI/BackupManager.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿/*
 TShock, a server mod for Terraria
-Copyright (C) 2011-2015 Nyx Studios (fka. The TShock Team)
+Copyright (C) 2011-2016 Nyx Studios (fka. The TShock Team)
```
Comment (nicatronTg) [inline TShockAPI/BackupManager.cs]: All the modified header files were changed in #1306, so this is unnecessary now. I would merge general-devel into your fork and not keep these superfluous changes.

File: TShockAPI/PlayerData.cs [hunk 2]
```diff
@@ -1,3 +1,21 @@
+/*
```
Comment (nicatronTg) [inline TShockAPI/PlayerData.cs]: Headers were also introduced in #1306. Remove extra headers too.

File: TShockAPI/Resources.Designer.cs [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
```
Comment (nicatronTg) [inline TShockAPI/Resources.Designer.cs]: This is auto generated and doesn't need a header.

**Comment by va6996** [comment]
Hope this fixes everything in #1305

**Comment by nicatronTg** [comment]
@va6996 yep! lgtm! Waiting on another person for PR review + merge and we're set. Thank you so much!

**Comment by Enerdy** [comment]
This is pretty good stuff.

**Pull request merged**